src/core/CL/kernels/CLPermuteKernel.cpp

```diff
@@ -56,10 +56,6 @@ Status validate_arguments(const ITensorInfo *input, const ITensorInfo *output, c
                                                          DataType::U16, DataType::S16,
                                                          DataType::U32, DataType::S32,
                                                          DataType::F16, DataType::F32);
-    ARM_COMPUTE_RETURN_ERROR_ON_MSG((perm != PermutationVector{ 2U, 0U, 1U })
```

Comment:
You should return error if the permutation vector size or the tensor dimensions are greater than 4

Reply:
Okay sure